components/brave_wallet/browser/hd_key.cc

```diff
@@ -166,6 +167,25 @@ std::string HDKey::GetPublicExtendedKey() const {
   return Serialize(MAINNET_PUBLIC, public_key_);
 }
 
+std::vector<uint8_t> HDKey::GetUncompressedPublicKey() const {
+  // uncompressed
```

Comment:
Will we support compressed? Just curious why one and not both?

Reply:
HDKey stores compressed public key and we don't need to access that directly, we provide `GetPublicExtendedKey` instead which is base58(hash160(public_key) 
Uncompressed public key is used only for eth address calculation. We want to restrict the access to `secp256k1_context` in the scope of HDKey so we don't want callers to get a compressed public key and covert it on their own.